core/experiments/manager.php

```diff
@@ -0,0 +1,309 @@
+<?php
+
+namespace Elementor\Core\experiments;
+
+use Elementor\Core\Base\Base_Object;
+use Elementor\Tools;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Manager extends Base_Object {
+
+	const STATUS_ALPHA = 1;
+
+	const STATUS_BETA = 2;
```

Comment:
`Alpah` & `Beta` are names of releases channels. so it better to use the `channel` term.
Also, please use strings to allow us to add `dev` and `rc` channels that they are before the alpha release.

```suggestion
	const CHANNEL_ALPHA = 'alpha';

	const CHANNEL_BETA = 'beta';
```

Reply:
Renamed to `RELEASE_STATUS_...`